utils/update.ts

```diff
@@ -0,0 +1,66 @@
+export * from "https://denopkg.com/hayd/deno-udd@master/mod.ts";
```

Comment:
Where this package is being used?

Reply:
Didn’t notice it was there, thanks, will remove!